### PR TITLE
New package: ccextractor-0.82

### PR DESCRIPTION
**Pull request by styrsven**

### CODE REVIEW

File: srcpkgs/ccextractor/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Template file for 'ccextractor'
+
+pkgname=ccextractor
+version=0.82
+revision=1
+wrksrc=${pkgname}.${version}
+only_for_archs="i686 x86_64"
```
Comment (ebfe) [inline srcpkgs/ccextractor/template]: Remove this so it's build for arm* and *-musl too.

File: srcpkgs/ccextractor/template [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+hostmakedepends="unzip"
+makedepends="leptonica-devel tesseract-ocr-devel"
+short_desc="Extract subtitles from video"
+maintainer="Styrsven <sven.a.jonsson@bahnhof.se>"
+license="GPL-2"
+homepage="http://www.ccextractor.org/doku.php"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}.src.${version}.zip"
+checksum=890e7786256c74c7e4850592784da027451dd7c3e3a353c9bad3bea5467b7b77
+
+pre_build() {
```
Comment (ebfe) [inline srcpkgs/ccextractor/template]: Maybe use post_extract here

File: srcpkgs/ccextractor/template [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	sed -i -e 's/\(^LDFLAGS \)=\(.*$\)/\1+=\2/' \
+	       -e 's/\(^CFLAGS \)=\(.*$\)/\1+=\2/' linux/Makefile
+}
+
+do_build() {
+	pushd linux
+	make
```
Comment (ebfe) [inline srcpkgs/ccextractor/template]: make CC=${CC} fixes the cross builds.

File: srcpkgs/ccextractor/template (hunk 2 above)
Comment (styrsven) [inline srcpkgs/ccextractor/template]: fixed

File: srcpkgs/ccextractor/template (hunk 3 above)
Comment (styrsven) [inline srcpkgs/ccextractor/template]: fixed

**Pull request merged**